packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -134,14 +134,9 @@ class ChromiumLauncher {
   final BrowserFinder _browserFinder;
   final Logger _logger;
 
-  bool get hasChromeInstance => _currentCompleter.isCompleted;
+  bool get hasChromeInstance => currentCompleter.isCompleted;
 
-  Completer<Chromium> _currentCompleter = Completer<Chromium>();
-
-  @visibleForTesting
-  void testLaunchChromium(Chromium chromium) {
-    _currentCompleter.complete(chromium);
-  }
+  Completer<Chromium> currentCompleter = Completer<Chromium>();
```

Comment:
This has to become a public API otherwise it is not possible to implement the API with a test class

Reply:
Can it be `@visibleForTesting`?

Reply:
Fixed